Neos.Flow/Tests/Behavior/Features/Bootstrap/SubProcess/SubProcess.php

```diff
@@ -81,10 +81,10 @@ public function quit()
     /**
      * Launch sub process
      *
-     * @return array The new sub process and its STDIN, STDOUT, STDERR pipes – or false if an error occurred.
+     * @return array|bool The new sub process and its STDIN, STDOUT, STDERR pipes – or false if an error occurred.
      * @throws \RuntimeException
      */
-    protected function launchSubProcess()
+    protected function launchSubProcess(): ?array
```

Comment:
Does that return type declaration work if a bool is returned? Seems weird to me.

Reply:
Nope. Overlooked the false option in return